cmd/docs.go

```diff
@@ -43,7 +43,11 @@ func runDocs(ctx *cli.Context) error {
 		// Clean up markdown. The following bug was fixed in v2, but is present in v1.
 		// It affects markdown output (even though the issue is referring to man pages)
 		// https://github.com/urfave/cli/issues/1040
-		docs = docs[strings.Index(docs, "#"):]
+		firstHashtagIndex := strings.Index(docs, "#")
+
+		if firstHashtagIndex != -1 {
```

Comment:
I would rather prefer a comparison to `>0`. This is simply due to it not being impossible, although highly unlikely, that one day, another default value could be chosen if no `#` is found (i.e. `-2`).
If the first char is a `#` (-> returned index is 0), then the slicing is unnecessary as well, so that way it also serves as a small optimization for edge cases.
```suggestion
		if firstHashtagIndex > 0 {
```

Reply:
> I would rather prefer a comparison to >0. This is simply due to it not being impossible, although highly unlikely, that one day, another default value could be chosen if no # is found (i.e. -2).

Hmm, well I'm interested in which scenario a non-found returned code is changed(and thus breaking Go-1 compatibility promise) 😄